mlflow/store/model_registry/file_store.py

```diff
@@ -905,6 +955,18 @@ def _read_helper(root, file_name, attempts_remaining=2):
 
         return _read_helper(root, file_name, attempts_remaining=retries)
 
+    def copy_model_version(self, mv, dst_name) -> ModelVersion:
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
```

Comment:
```suggestion
        :param mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
```

Reply:
Will do and apply across the different APIs.

Reply:
Just to be clear @harupy do you prefer `mv` as a param name? I have `src_mv` in other `copy_model_version` definitions.

Reply:
I would use `src_model_version` :)

Reply:
`src_mv` is fine. This isn't exposed to users.